v2/cli/init_templates.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"bytes"
+	"text/template"
+
+	"github.com/Masterminds/sprig"
+)
+
+// Template project.yaml file for a non-git project
+// nolint: lll
+var projectTemplate = []byte(`# yaml-language-server: $schema=https://raw.githubusercontent.com/brigadecore/brigade/v2/v2/apiserver/schemas/project.json
+apiVersion: brigade.sh/v2-alpha.5
+kind: Project
+metadata:
+  id: {{ .ProjectID }}
+description: My new Brigade project
+spec:
+  eventSubscriptions:
+  - source: brigade.sh/cli
+    types:
+    - exec
+  workerTemplate:
+    # logLevel: DEBUG
+    {{- if eq .GitCloneURL "" }}
+    defaultConfigFiles:
+      brigade.{{ .Language }}: |
+        {{ nindent 8 .Script }}
+    {{- else }}
+    git:
+      cloneURL: {{ .GitCloneURL }}
+    {{- end }}
+`)
+
+// nolint: lll
+var scriptTemplate = []byte(`{{- if eq .Language "js" -}}
+const { events } = require("@brigadecore/brigadier");
+
+events.on("brigade.sh/cli", "exec", async event => {
+	let job = new Job("hello", "debian:latest", event);
+	job.primaryContainer.command = ["echo"];
+	job.primaryContainer.arguments = ["Hello, World!"];
+	await job.run();
+});
+
+events.process(); 
+{{- else -}}
+import { events, Job } from "@brigadecore/brigadier"
+
+events.on("brigade.sh/cli", "exec", async event => {
+	let job = new Job("hello", "debian:latest", event)
+	job.primaryContainer.command = ["echo"]
+	job.primaryContainer.arguments = ["Hello, World!"]
+	await job.run()
+})
```

Comment:
Let's split this into two separate templates and decide which template to use in the code that renders a script from the template.

This isn't terrible as is, but I'm aware of the probability of adding support for a third "script" style (declarative YAML) that will likely land in 2.1. Based on that, I'd say _one_ script template like we have now is going to become unwieldy quickly if the template keeps growing with additional conditional logic.